### DISCUSSION TITLE
All code in appendix using knitr::all_labels()

### DISCUSSION THREAD

**Discussion by dmenne** [quarto-dev/quarto-cli#3290] [Q&A]
[Yihui](https://bookdown.org/yihui/rmarkdown-cookbook/code-appendix.html#code-appendix) describes a way to use `knitr::all_labels()` to show the code of a chunks in an appendix.

I understand why this does not work in Quarto, but is there a workaround?

**Comment by mcanouil** [comment]
FYI:
- https://github.com/quarto-dev/quarto-cli/discussions/6650#discussioncomment-6861503
